docker-compose.yml

```diff
@@ -145,8 +148,13 @@ services:
       - "8080"
 
   chrome:
-    image: selenium/standalone-chrome:3.141
-    shm_size: 2G
+    image: selenium/standalone-chrome-debug:3.141.59-20200525
+    volumes:
+      - /dev/shm:/dev/shm
+    shm_size: 2 
+    ports:
+      - "4444:4444"
+      - "5959:5900"
```

Comment:
do you need these port bindings

Reply:
These port bindings let me use screen sharing tool to watch the specs go through the ui.

Reply:
that sounds pretty cool. maybe a corresponding snippet in the readme?